account_easy_reconcile/__openerp__.py

```diff
@@ -54,7 +55,6 @@
 """,
```

Comment:
Nevertheless, if you can come up with a summary, you can add a short one line description under a `'summary'` key